espnet2/asr_transducer/beam_search_transducer.py

```diff
@@ -0,0 +1,711 @@
+"""Search algorithms for Transducer models."""
```

Comment:
Is there a plan to support decoding in batches?

I see that implementation in this file supports only
decoding a single utterance at a time.

Reply:
There was a plan to make a unified `ScorerInterface` and `BatchScorerInterface` with other models but we faced some difficulties. Supporting batch decoding is on my to do, I'll revisit it after everything related to this version is sorted!